eodatasets/prepare/ls_usgs_l1_prepare.py

```diff
@@ -284,7 +288,8 @@ def find_gz_mtl(ds_path, output_folder):
         members = [m for m in tar_gz.getmembers() if mtl_pattern.search(m.name)]
         if not members:
             return None
-        tar_gz.extractall(output_folder, members)
+        # extractall requires the output location as a string
```

Comment:
I'm surprised the tests didn't fail -- they extract a tar. Different Python version?

The comment probably isn't necessary for future readers: we do the same str/path conversion in lots of places (it could be relevant in the commit message though, if this fix was alone).

Reply:
I was testing it in the NCI environment; just using setup.py to install.